src/main.rs

```diff
@@ -156,7 +165,9 @@ fn main() -> CliResult {
             Box::new(writer)
         }
     };
-    generate_xml(&mut out, reader, &args.category, args.record_type)?;
+    let num_records = generate_xml(&mut out, reader, &args.category, args.record_type)?;
+    std::mem::drop(progress_bar);
```

Comment:
Here we probably should have a comment, telling that we drop progress bar, so it's removed from the stderr, before we print the performance metrics (otherwise the drop handler would erroneously remove the lower lines of the performance metrics output).